django==4.0.4
djangorestframework
djangochannelsrestframework
channelsmultiplexer
django-cors-headers
drf-writable-nested
channels-redis==3.4.1 # Version 4.0.0 appears to have a problem with "redis.exceptions.ConnectionError: max number of clients reached"
cryptography
pynacl
dulwich

python-can
gs_usb
python-can[gs_usb]
pyusb
pyserial
libusb
uptime
websockets
psutil
requests
cachetools

# meta (building / packaging / testing / ...)
pytest-django
pyinstaller>=5.12
mypy
tox
pip-licenses
pipdeptree

# HydraVision
djangorestframework-simplejwt
influxdb_client